.env

```diff
@@ -12,5 +12,5 @@ NEXT_PUBLIC_ALGOLIA_SEARCH_ONLY_API_KEY=2cd9898a6c253bfa3965d2b62a4f7f3d
 MKTG_CONTENT_API="https://content.hashicorp.com"
 # Note: check .env.local.example for additional required env vars
 
-NEXT_PUBLIC_LEARN_API_BASE_URL="https://pidr8rzmwp.us-east-1.awsapprunner.com"
+NEXT_PUBLIC_LEARN_API_BASE_URL="https://ycmsw3yg4a.us-east-1.awsapprunner.com"
```

Comment:
So we have this configured in vercel as well for preview and local, I'm contemplating just deleting this 🤔